primitives/io/src/lib.rs

```diff
@@ -333,8 +401,8 @@ pub trait DefaultChildStorage {
 		let child_info = ChildInfo::new_default(storage_key);
 		let (all_removed, num_removed) = self.kill_child_storage(&child_info, limit);
 		match all_removed {
-			true => KillStorageResult::AllRemoved(num_removed),
-			false => KillStorageResult::SomeRemaining(num_removed),
+			true => KillStorageResult::AllRemoved { backend: num_removed, overlay: 0 },
```

Comment:
why is it written that 0 keys are removed from the overlay ?

Reply:
Oh...my bad
Will fix this